yarn-project/end-to-end/src/sample-dapp/index.test.mjs

```diff
@@ -1,25 +1,36 @@
 import { createSandbox } from '@aztec/aztec-sandbox';
-import { Contract, createAccount } from '@aztec/aztec.js';
-import { PrivateTokenContractAbi as PrivateTokenArtifact } from '@aztec/noir-contracts/artifacts';
+import { Contract, Fr, computeMessageSecretHash, createAccount } from '@aztec/aztec.js';
+import { TokenContractAbi } from '@aztec/noir-contracts/artifacts';
 
-describe('private token', () => {
+describe('token', () => {
   // docs:start:setup
   let rpc, stop, owner, recipient, token;
   beforeAll(async () => {
     ({ rpcServer: rpc, stop } = await createSandbox());
     owner = await createAccount(rpc);
     recipient = await createAccount(rpc);
-    token = await Contract.deploy(owner, PrivateTokenArtifact, [100n, owner.getAddress()]).send().deployed();
+
+    token = await Contract.deploy(owner, TokenContractAbi, []).send().deployed();
+    await token.methods._initialize({ address: owner.getAddress() }).send().wait();
```

Comment:
hmm, we might want to move the docs:start:setup to another example now, as this has rough syntax with the initialise

Reply:
Think it might be fine to point out that you need to initialise if you need any private values set etc, and that it is horribly insecure but it will be addressed when constructor can do calls.